calico/getting-started/kubernetes/managed-public-cloud/aks.md

```diff
@@ -43,7 +43,7 @@ The geeky details of what you get:
     # Create a resource group
     az group create --name my-calico-rg --location westcentralus
 
-    az aks create --resource-group my-calico-rg --name my-calico-cluster --location westcentralus --network-plugin none
+    az aks create --resource-group my-calico-rg --name my-calico-cluster --location westcentralus --pod-cidr 192.168.0.0/16 --network-plugin none
```

Comment:
Have you checked that this doesn't cause problems being mis-matched with the kube-proxy CIDR?

Reply:
Yes. This option allows us to set kube-proxy config to `192.168.0.0/16` whereas it was `10.244.0.0/16` without it.

```
      containers:
      - command:
        - kube-proxy
        - --metrics-bind-address=0.0.0.0:10249
        - --kubeconfig=/var/lib/kubelet/kubeconfig
        - --cluster-cidr=192.168.0.0/16
        - --v=3
```